app/mockServices/mockData/openshift-images.ts

```diff
@@ -21,7 +21,7 @@ export const imagesData = {
                     'description': 'Provides a Jenkins 1.X server on CentOS 7. For more information about using this container image, including OpenShift considerations, see https://github.com/openshift/jenkins/blob/master/README.md.',
                     'iconClass': 'icon-jenkins',
                     'openshift.io/display-name': 'Jenkins 1.X',
-                    'tags': 'jenkins',
+                    'tags': 'builder,jenkins',
```

Comment:
Locally, I removed the 'builder' tag from Jenkins and added a Jenkins serviceClass instead.  Let me know if you prefer that setup/mock data instead. -thanks